kusto/ingest/internal/filesystem/filesystem.go

```diff
@@ -34,8 +34,8 @@ const (
 	// made a 5x improvement in speed. We don't have any numbers from the service side to give us numbers we should use, so this
 	// is our best guess from observation. DO NOT CHANGE UNLESS YOU KNOW BETTER.
 
-	blockSize   = 8 * _1MiB
-	concurrency = 50
+	BlockSize   = 8 * _1MiB
+	Concurrency = 50
```

Comment:
These should stay private.  We can document in the constructor options what the default values are.

Reply:
These didn't get marked as private, still are public.